src/pages/LandingPages/MGCovid19/MGCovidHero.vue

```diff
@@ -119,8 +119,8 @@ export default {
 	computed: {
 		pageCopy() {
 			return {
-				headline: 'Join Kiva\'s Global <span class="no-wrap">COVID—19</span> Response', // eslint-disable-line max-len
-				subhead: 'Lend a hand to provide relief to entrepreneurs and small businesses impacted <span class="no-wrap">by COVID—19.</span>', // eslint-disable-line max-len
+				headline: 'Join Kiva\'s Global <span class="no-wrap">COVID‑19</span> Response', // eslint-disable-line max-len
+				subhead: 'Lend a hand to provide relief to entrepreneurs and small businesses impacted <span class="no-wrap">by COVID‑19.</span>', // eslint-disable-line max-len
```

Comment:
Please note the original `dash` character would prevent the break of the Covid-19 on it's own without the no-wrap span. Previously we had other words in side the no-wrap span which I had to move out to prevent them word "Response" from falling off of the page to the right in mobile.

Reply:
https://www.toptal.com/designers/htmlarrows/punctuation/non-breaking-hyphen/